docs/proposals/kubelet-eviction.md

```diff
@@ -69,6 +69,7 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
 | Eviction Signal  | Description                                                                     |
 |------------------|---------------------------------------------------------------------------------|
 | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
+| disk.available   | disk.available := node.stats.fs.available |
```

Comment:
The input we take specifies the **minimum availability**, which is
unrelated to the partition size... So if a user sets
`--hard-eviction-threshold=disk.available<1GB`, that would apply to both
partitions. It does not matter how much space there is on each partition.
Are you thinking of filesystems becoming slower when they are close to
their capacity? If so, do you know if that problem is proportional to the
disk capacity or is it more or less a constant number?

On Fri, May 6, 2016 at 4:49 PM, Dawn Chen <notifications@github.com> wrote:

> In docs/proposals/kubelet-eviction.md
> <https://github.com/kubernetes/kubernetes/pull/25237#discussion_r62405214>
> :
>
> > @@ -69,6 +69,7 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
> >  | Eviction Signal  | Description                                                                     |
> >  |------------------|---------------------------------------------------------------------------------|
> >  | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
> > +| disk.available   | disk.available := node.stats.fs.available |
>
> Unifying image and root filesystem threshlods might not very useful if the
> threshold is hardcoded number, instead of a configurable percentage (%).
> Normally you have a big partition for your image, and small partition for
> your root. Also normally image size is relatively big, evicting image might
> resolve out-of-disk.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25237/files/e7053d1db93e2b92f27a777bc230a6a8ed901cba#r62405214>
>